### ISSUE TITLE
Investigate frontend-app-learning as optional for Lilac

### ISSUE THREAD

**Issue by BbrSofiane** [openedx/build-test-release-wg#45]
Investigate frontend-app-learning as optional for Lilac

**Comment by cmltaWt0** [comment]
First attempt to make it works is partially successful - I can deploy, configure it, be redirected to the learning mfe.

In the app I can:
1. See the Course structure
2. Bookmart unit
3. Move between units

I fail with loading a content of the unit though
<img width="1631" alt="image" src="https://user-images.githubusercontent.com/877401/116259390-6e1c5300-a77e-11eb-8581-bd5f11d1529f.png">


The only different with default MFEs configuration is in domain - I'm using `app-lilac-*.xxx.com` for mfes and `las-lilac-*.xxx.com` for lms.
In this configuration account, profile and grade book mfes work fine.


Also there are no failed network XHR requests
<img width="816" alt="image" src="https://user-images.githubusercontent.com/877401/116260315-3e217f80-a77f-11eb-9794-4c57273e6981.png">

**Comment by cmltaWt0** [comment]
@davidjoy @BbrSofiane 

I've found that Unit content is successfully loaded to the page but is not resized in my case. 
So I've emulated this resize event manually and got Unit content visible.
<img width="1634" alt="image" src="https://user-images.githubusercontent.com/877401/116735732-9b276a80-a9f7-11eb-922e-77e650fb0269.png">

I believe I'll find the root cause of this but I'll be appreciated if anyone throw me an idea why `plugin.resize` event doesn't happen.
